nextcloud/docker-compose.yml

```
@@ -51,7 +52,7 @@ services:
       - redis
```

Cannot mark the whole depends_on here for some reason. But that is how it should look like.
```suggestion
    depends_on:
      db:
        condition: service_healthy
      redis:
        condition: service_started
```